docs/pull-requests.rst

```diff
@@ -59,17 +59,15 @@ Troubleshooting
 #. **Pull Requests builds are not triggered**.
    We only support GitHub and GitLab currently.
    You need to make sure that your Read the Docs account is connected with that provider.
-   You can check this by going to your `profile settings`_.
+   You can check this by going to your :guilabel:`Username dropdown > Settings > Connected Services`.
```

Comment:
```suggestion
   You can check this by going to your :guilabel:`Username dropdown` > :guilabel:`Settings` > :guilabel:`Connected Services`.
```

Just to keep consistency. We are splitting each "click required" into a guilabel 😄